packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -25,9 +29,10 @@ export type RenderProps = {|
     multipleSelect?: boolean,
     countChoices?: boolean,
     deselectEnabled?: boolean,
-    choices: $ReadOnlyArray<PerseusRadioChoice>,
+    choices: $ReadOnlyArray<RadioChoiceWithMetadata>,
     selectedChoices: $ReadOnlyArray<PerseusRadioChoice["correct"]>,
-    choiceStates?: $ReadOnlyArray<$FlowFixMe>,
+    choiceStates?: $ReadOnlyArray<ChoiceState>,
+    values?: $ReadOnlyArray<boolean>,
```

Comment:
What's this change for? If I'm understanding correctly, this change changes the contract with the consumer and the library right? It makes me a little nervous, so I would double check this if you haven't already.

Reply:
I paired with Tamara on this. This is existing behavior - support for a legacy way of handling changes? we didn't feel confident in pulling out that logic so we added the prop types that were being used but weren't previously defined

Reply:
We could use that previous pattern you liked Matthew, and create a new type for just this file using the Props type and adding values to that. Not sure if that would make things more cluttered. And since values is actually accessed from Props here, maybe it is necessary?

Reply:
Oh right, I remember this now. Let's comment it - marking it as depreciated and adding these notes on what it was for and why we kept it.